backend/tests/unit/dal/Services/PropertyServiceTest.cs

```diff
@@ -0,0 +1,173 @@
+using FluentAssertions;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Moq;
+using Pims.Core.Test;
+using Pims.Dal.Entities;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Pims.Dal.Services;
+using Xunit;
+using Microsoft.EntityFrameworkCore;
+using Pims.Core.Extensions;
+
+namespace Pims.Dal.Test.Services
+{
+    [Trait("category", "unit")]
+    [Trait("category", "dal")]
+    [Trait("group", "property")]
+    [ExcludeFromCodeCoverage]
+    public class PropertyServiceTest
+    {
+        #region Tests
+        #region GetById
+        [Fact]
+        public void GetById_Success()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission(Permissions.PropertyView);
+
+            var property = EntityHelper.CreateProperty(1);
+            helper.CreatePimsContext(user, true).AddAndSaveChanges(property);
```

Comment:
if we are testing the service, is the context necessary?

Reply:
I guess we might not need it but I followed other existing tests that do that. Wouldn't want to go a different way and find that it doesn't work. In any case, do you see harm in that?

Reply:
Is just that any code that does not add value should be removed if possible.